src/drawer/components/DrawerCard/UserDrawerCard/UserDrawerCard.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useRef, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Card } from '@/components/Card/Card';
+import { Dropdown } from '@/components/Dropdown/Dropdown';
+
+import { DrawerCardProps } from '../DrawerCard.type';
+
+import {
+  StyledServiceModify,
+  StyledServiceText,
+  StyledServiceTextContainer,
+} from './UserDrawerCard.style';
+
+export const UserDrawerCard = ({
+  link,
+  bigImgSrc,
+  smallImgSrc,
+  title,
+  body,
+  bookmarkCount,
+  isBookmarked,
+}: DrawerCardProps) => {
+  const [isCardSettingClicked, setIsCardSettingClicked] = useState(false);
+  const containerRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (containerRef.current && !containerRef.current.contains(event.target as Node)) {
+        setIsCardSettingClicked(false);
+      }
+    };
+
+    document.addEventListener('mousedown', handleClickOutside);
+
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, []);
+
+  const onClickSetting = () => {
+    setIsCardSettingClicked((prev) => !prev);
+  };
+
+  return (
+    <Card link={link}>
+      <Card.BigThumbnail imgSrc={bigImgSrc} />
+      <Card.SmallThumbnail imgSrc={smallImgSrc} />
+      <Card.Content
+        title={title}
+        body={body}
+        bookmarkCount={bookmarkCount}
+        isBookmarked={isBookmarked}
+      />
+      <DropdownContainer ref={containerRef}>
+        <Card.Setting onClick={onClickSetting} />
+        {isCardSettingClicked && (
+          <Dropdown padding="1rem" bottom="-4rem" right="0">
```

Comment:
피그마 확인해보니까 세팅 버튼을 가리지 않더라고요!!!!
bottom 값만 살짝 줄여봤어요 괜찮으면 아래 제안에서 'commit suggestion' 눌러서 반영 부탁드립니더 ㅎㅅㅎ

```suggestion
          <Dropdown padding="1rem" bottom="-4.5rem" right="0">
```

|지금|피그마|bottom 줄였을 때|
|---|---|---|
|![image](https://github.com/yourssu/Soomsil-Web/assets/87255462/509ad9b4-07ce-45d0-be40-61000824fdbe)|![image](https://github.com/yourssu/Soomsil-Web/assets/87255462/31cda428-03f1-4952-ac60-04e2ba6f254a)|![image](https://github.com/yourssu/Soomsil-Web/assets/87255462/9d7faedd-c7c0-4948-b8a4-04868b9a093a)|

피그마 사진 왜이렇게 작은........................... 체리눈절대지켜............

Reply:
악. 그리고 드롭다운 스타일에 관해 질문이 하나 있는데요...!
홈에서의 드롭다운은 배경색 변화가 없지만, 서랍장에서의 드롭다운은 열렸을 때 배경색이 약간 어두워지더라고요..!
(헤더, UserDrawerCard의 드롭다운 모두)

이 부분을 어떻게 처리해야 할 지 저도 먼가........ 생각을 좀 해보겠습니다.............
당장은 괜찮은 방법이 안 떠오르네요 ╯︿╰

Reply:
오잉 저는 배경색도 안 어두워지고 세팅 버튼도 안 가리는 상태라,,,,,, bottom 값을 4.5로 변경하니 너무 떨어져 보이는 상태네요 ㅠㅡㅠ
이것도 해상도마다 다른 것 같아요,,,,,,,,
![image](https://github.com/yourssu/Soomsil-Web/assets/84809236/dfe8b7d7-c2a1-41d4-8b3d-9390e98d7961)

Reply:
프리뷰로 보셔야합니다~!
![example2](https://github.com/yourssu/Soomsil-Web/assets/87255462/e39e2445-f586-4642-9d8d-7008f850afe5)

Reply:
아아아아아아 피그마를 얘기하시는 거였군요!!! 저는 채리 코드 얘기하시는 줄 알았슴다....ㅎ
이거 Radix 쓰면 기본적으로 제공하는 옵션인데 고민되네용 흠 ㅠㅡㅠ

Reply:
bottom="-4.5rem" 커밋 왼료 입니다~ 감사해요 짱보리><

흠 다른 팀에서 드롭다운 사용하실 때두 저렇게 어두워지는 거면 공통컴포넌트인 드롭다운 파일 쪽에서 뒷배경 처리하도록... 해야할 것 같긴 한데 한 번 다같이 이야기해볼 필요가 있을 것 같아욥

Reply:
이게..... 또 홈에서는 배경색 ❌인데 서랍장에서 ⭕️에요 엉 엉

Reply:
근데 요거는 디자인 팀에서도 서랍장이랑 홈이랑 얘기 안 해보시지 않았을까욥...?? 오늘 여쭤봐도 좋을 것 같슴다!!

Reply:
확인이욥ㅎㅎㅎ